### DISCUSSION TITLE
dpgen's fp process stucked

### DISCUSSION THREAD

**Discussion by theAfish** [deepmodeling/dpgen#491] [Q&A] [answered]
Hi all,

I am using vasp on a server to doing the fp part of dpgen. However, the vasp job was stuck and cannot enter the SCF process. 
![微信截图_20210808120530](https://user-images.githubusercontent.com/85424334/128620341-95d1eeb2-7521-4559-92c6-e7c68d65af48.png)

Meanwhile, if I use my own script with the auto-generated input files by dpgen, the calculation goes well. Here is my script:
 
![image](https://user-images.githubusercontent.com/85424334/128620381-b7227777-98df-4c0a-a904-e5d4d2449577.png)

I have also changed the pbs.py of dpdispatcher, but it still does not work. the front part of auto-generated xxx.sub file of adjusted dpgen is:
![image](https://user-images.githubusercontent.com/85424334/128620406-443af590-150d-41f3-819b-18ba4dc39758.png)

Could someone explain what's happening? Thank you very much

**Comment by AnguseZhang** [comment, marked as answer]
It seems that there is some mismatching between your cluster and the scripts dpdispatcher generates. 
I think your direction is correct.  Just compare the scripts carefully and see which command is invalid. If you have questions about any command, we can help to explain.

**Reply by theAfish** [reply]
Thank you very much! I was wondering if there was a part of this script that would jam the calculation process. However, I don't think the script itself should have any impact on the calculation. Still not work

**Comment by theAfish** [comment]
I found that the script works well if I only allow one job to be calculated at a time. So this could be something strange in my parallel setups. (However, if I submit multiple tasks manually with my own script, it doesn't cause any problems)
